app/models/billboard.rb

```diff
@@ -214,10 +216,11 @@ def generate_billboard_name
   def process_markdown
     return unless body_markdown_changed?
 
-    if FeatureFlag.enabled?(:consistent_rendering)
```

Comment:
I would like to remove the feature flag and use "consistent rendering" by default (for other models, like Articles).

Reply:
Yeah @lightalloy I was thinking the same thing, and having this be a part of that.

I made an issue for the broader task:

https://github.com/forem/forem/issues/20289

Labelled available for external, but grab it yourself if you're interested.